app-crypt/codecrypt/codecrypt-1.8.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="Post-quantum cryptography tool"
+DESCRIPTION="GnuPG-like unix program for encryption and signing that uses only \
+quantum-computer-resistant algorithms"
+HOMEPAGE="http://e-x-a.org/codecrypt"
+COPYRIGHT="2013-2016 Mirek Kratochvil"
+LICENSE="
+	GNU GPL v3
+	GNU LGPL v3
+	"
+REVISION="1"
+SOURCE_URI="http://e-x-a.org/codecrypt/files/codecrypt-$portVersion.tar.gz"
+CHECKSUM_SHA256="3be7c2a548aa336dbac706c3cab2071664d02b11652c16c02a8b4fb5cac6bc88"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	codecrypt$secondaryArchSuffix = $portVersion
+	cmd:ccr$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcryptopp$secondaryArchSuffix
+	lib:libfftw3$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcryptopp$secondaryArchSuffix
+	devel:libfftw3$secondaryArchSuffix
+	devel:libgmp$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:cmp
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	CPPFLAGS="-D_BSD_SOURCE" LIBS="-lbsd" runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+}
```

Comment:
I think it would be nice to ship the README. What do you think?
```diff
 	make install
+	install -d "$docDir"
+	install -t "$docDir" README
```

I'll try building everything on x86_gcc2 for the x86 secondary arch.

BTW, for recipes which are not available on x86_gcc2 primary arch and which, consequently, need to be built for the x86 secondary arch, we sometimes choose to put the runtime in $prefix/bin instead of $binDir.
This is done by defining commandBinDir and commandSuffix, as you can see in e.g. app-office/calcurse/calcurse-4.3.0.recipe.

I don't know if it would be worth making the same changes for this recipe.
The only advantage would be that, on x86_gcc2, instead of typing ccr-x86 we would be able to type just ccr.

Let's leave codecrypt as it is for now, and, if other mentors/developers ask for the commandBinDir-commandSuffix "trick" before we merge, then, if you feel like doing it, that's ok for me.